Resources/doc/2-the-view-layer.rst

```diff
@@ -88,10 +94,12 @@ or it is possible to use lazy-loading:
 
     <?php
 
-    use FOS\RestBundle\Controller\FOSRestController;
+    use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 
-    class UsersController extends FOSRestController
+    class UsersController extends Controller
     {
+        use ContainerAwareControllerTrait;
```

Comment:
You should not use the ``ContainerAwareTrait`` apparently as it is already used by ``Controller``.